### PR TITLE
Added option to configure the grpc histogram

### PR DESCRIPTION
**Pull request by cristiancl25**

### CODE REVIEW

File: mlserver/settings.py [hunk 1]
```diff
@@ -190,6 +191,9 @@ class Config:
     grpc_max_message_length: Optional[int] = None
     """Maximum length (i.e. size) of gRPC payloads."""
 
+    grpc_response_latency_histogram : bool = False
```
Comment (adriangonz) [inline mlserver/settings.py:194]: Would there be any caveat to removing this setting and always keeping it enabled? It seems reasonable (and aligns it with its REST counterpart - where I think the latency histogram is also always enabled).

**Review by adriangonz — APPROVED** [review]
Thanks for the changes @cristiancl25 !

PR looks great now. Once tests are green, we can go ahead with this one. :rocket:

**Comment by cristiancl25** [comment]
Done!

**Comment by adriangonz** [comment]
Thanks for rebasing it! It should be ready to go now  :+1:

**Pull request merged**